app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    try:
+        for friend in friends:
+            try:
+                cafe.visit_cafe(friend)
+            except NotWearingMaskError:
+                masks_to_buy += 1
+            except VaccineError:
+                raise
```

Comment:
You don't need to re-raise exception here. Since your `VaccineError` instances have message you have to return, you can do this:
```python
except VaccineError as e:
    return str(e)
```
and get rid of the outer `try ... except` block.

Reply:
Thanks, for the good advice!